src/screens/VocabScreen.js

```diff
@@ -0,0 +1,112 @@
+/**
+  Map screen.
+
+*/
+
+import type {
+  NavigationScreenConfig,
+  NavigationScreenProp,
+  NavigationState,
+} from 'react-navigation';
+
+import type { VocabEntry } from 'src/entities/Types';
+
+import type { GameStoreProps } from 'src/undux/GameStore';
+
+import React from 'react';
+
+import {
+  ActivityIndicator,
```

Comment:
Remove any `import`s that aren't being used. The linter should highlight unused imports for you.